src/algorithm/core.py

```diff
@@ -754,38 +754,40 @@ def draw_marked_bubbles_with_evaluation_meta(
 
     def draw_evaluation_summary(self, marked_image, evaluation_meta, evaluation_config):
         # TODO: update this condition
-        if evaluation_config.draw_answers_summary:
+        if evaluation_config.draw_answers_summary['enabled']:
             self.draw_answers_summary(
                 marked_image, evaluation_config, evaluation_meta["score"]
             )
         # TODO: update this condition
-        if evaluation_config.draw_score:
+        if evaluation_config.draw_score['enabled']:
             self.draw_score(marked_image, evaluation_config, evaluation_meta["score"])
         return marked_image
 
     def draw_answers_summary(self, marked_image, evaluation_config, score):
-        h, w = marked_image.shape[:2]
-
+        # h, w = marked_image.shape[:2]
+        summary_position = evaluation_config.draw_answers_summary['position']
         # TODO: pickup position from evaluation_config
         formatted_answers_summary = evaluation_config.get_formatted_answers_summary()
-
-        # Draw the aggregate answers summary
-        summary_position = lambda size_x, size_y: (
-            w - size_x - w // 20,
-            h // 30,
-        )
-        ImageUtils.draw_text(marked_image, formatted_answers_summary, summary_position)
+        
+        text_size=evaluation_config.draw_answers_summary['size']
+        ImageUtils.draw_text(marked_image,formatted_answers_summary,summary_position,text_size=text_size,thickness=int(text_size*2))
+        
 
     def draw_score(self, marked_image, evaluation_config, score):
         # TODO: pickup from evaluation_config using format string/eval
-        formatted_score, score_position = evaluation_config.get_formatted_score(score)
-
+        formatted_score = evaluation_config.get_formatted_score(score)
+        score_position=evaluation_config.draw_score['position']
         # Draw the final score
+        
+        text_size=evaluation_config.draw_score['size']
```

Comment:
let's keep this inside get_formatted_score function and return if from there as three values
```suggestion
        formatted_score, score_position, text_size, thickness = evaluation_config.get_formatted_score(score)
```